utils/utils.py

```diff
@@ -286,10 +288,15 @@ def get_trained_models(log_folder: str) -> Dict[str, Tuple[str, str]]:
     for algo in os.listdir(log_folder):
         if not os.path.isdir(os.path.join(log_folder, algo)):
             continue
-        for env_id in os.listdir(os.path.join(log_folder, algo)):
+        for model_folder in os.listdir(os.path.join(log_folder, algo)):
```

Comment:
Please rather rely on the `args.yml`  file, monitor file may not be present and feels very hacky...

For the `Pendulum-v1` env that produces errors, please submit a PR to the rl-trained-agents repo ;)

I will probably need to manually update the hugginface models too

Reply:
Fixed it.